src/server/game/Movement/MotionMaster.h

```diff
@@ -212,27 +165,26 @@ class TC_GAME_API MotionMaster //: private std::stack<MovementGenerator *>
         void MovePath(uint32 path_id, bool repeatable);
         void MoveRotate(uint32 time, RotateDirection direction);
 
-        MovementGeneratorType GetCurrentMovementGeneratorType() const;
-        MovementGeneratorType GetMotionSlotType(int slot) const;
-
-        void propagateSpeedChange();
-
-        bool GetDestination(float &x, float &y, float &z);
     private:
-        void Mutate(MovementGenerator *m, MovementSlot slot);                  // use Move* functions instead
+        bool NeedInitTop() const;
+        void InitTop();
+        void Pop();
+
+        void Mutate(MovementGenerator *m, MovementSlot slot);
 
         void DirectClean(bool reset);
         void DelayedClean();
-
         void DirectExpire(bool reset);
         void DelayedExpire();
-
-        typedef std::vector<_Ty> ExpireList;
-        ExpireList* _expList;
-        _Ty Impl[MAX_MOTION_SLOT];
-        int _top;
-        Unit* _owner;
-        bool _needInit[MAX_MOTION_SLOT];
-        uint8 _cleanFlag;
+        void DirectDelete(MovementGenerator* curr);
+        void DelayedDelete(MovementGenerator* curr);
+
+        ExpireList* m_expireList;
+        MovementGenerator* m_slot[MAX_MOTION_SLOT];
+        int m_top;
+        Unit* m_owner;
+        bool m_initialize[MAX_MOTION_SLOT];
+        uint8 m_cleanFlag;
```

Comment:
well there are 4 ways of naming used in the core
* m_myVariable
* myVariable
* _myVariable
* MyVariable

I just used the most common one in core classes